ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml

```diff
@@ -806,6 +806,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-workers-rhel8,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
why is this true in 4.9?

Reply:
by default it's true and looks like it's that way from 4.9 and lower. here is the [commit](https://github.com/openshift/release/commit/a2db9bce045) that changed it to false.

I guess the idea is that we can skip these kinds of jobs if it's a change to something unrelated to actual functional code.
I'm not sure why it isn't there for 4.9 and lower though. if we want to do that, I can do it in a different PR.

Reply:
Can you please update this to `always_run: false` here to match the rest of the PR.

Reply:
ok, added a new commit to this PR to take care of that.